PullRequestCreated/PullRequestCreated.ts

```diff
@@ -45,7 +55,7 @@ class PullRequestCreated extends OctokitAction {
   }
 
   private findMentionedIssues(pr: PullRequest): string[] {
-    return pr.body?.match(JIRA_ISSUE_PATTERN) || [];
+    return [...new Set(pr.body?.match(JIRA_ISSUE_PATTERN) || [])];
```

Comment:
`.Distinct()`

Reply:
It happened to me after adding the linked issue:
```
[PAVEL-1](https://sonarsource.atlassian.net/browse/PAVEL-1)
```

The issue will be found twice.

Reply:
About not using Set directly, AFAIK, you cannot use it as an array.

Reply:
All we need is to iterate it

Reply:
The idea was to change return type and parameter of the `firstNonSubTask` to the Set itself. If possible.